src/components/elements/MapPolygonPanel/AttributeInformation.tsx

```diff
@@ -1,75 +1,167 @@
 import { useT } from "@transifex/react";
+import { useEffect, useState } from "react";
 
+import Button from "@/components/elements/Button/Button";
 import Dropdown from "@/components/elements/Inputs/Dropdown/Dropdown";
 import Input from "@/components/elements/Inputs/Input/Input";
+import { useMapAreaContext } from "@/context/mapArea.provider";
+import { fetchPutV2TerrafundSitePolygonUuid, useGetV2TerrafundPolygonUuid } from "@/generated/apiComponents";
+import { SitePolygon } from "@/generated/apiSchemas";
 
 import Text from "../Text/Text";
 
 const dropdownOptionsRestoration = [
   {
     title: "Tree Planting",
-    value: 1
+    value: "Tree Planting"
   },
   {
     title: "Direct Seeding",
-    value: 2
+    value: "Direct Seeding"
   },
   {
     title: "Assisted Natural Regeneration",
-    value: 3
+    value: "Assisted Natural Regeneration"
   }
 ];
 const dropdownOptionsTarget = [
   {
     title: "Agroforest",
-    value: 1
+    value: "Agroforest"
   },
   {
     title: "Natural Forest",
-    value: 2
+    value: "Natural Forest"
   },
   {
     title: "Mangrove",
-    value: 3
+    value: "Mangrove"
   },
   {
     title: "Peatland",
-    value: 4
+    value: "Peatland"
   },
   {
     title: "Riparian Area or Wetland",
-    value: 5
+    value: "Riparian Area or Wetland"
   },
   {
     title: "Silvopasture",
-    value: 6
+    value: "Silvopasture"
   },
   {
     title: "Woodlot or Plantation",
-    value: 7
+    value: "Woodlot or Plantation"
   },
   {
     title: "Urban Forest",
-    value: 8
+    value: "Urban Forest"
   }
 ];
 
 const dropdownOptionsTree = [
   {
     title: "Single Line",
-    value: 1
+    value: "Single Line"
   },
   {
     title: "Partial",
-    value: 2
+    value: "Partial"
   },
   {
-    title: "Full",
-    value: 3
+    title: "Full Coverage",
+    value: "Full Coverage"
```

Comment:
If I'm reading this correctly, the titles for these drop down options aren't getting translated. It's a little messy, but the first thing that comes to mind for me to manage that would be create a hook that maps these option lists in a `useMemo()` and uses `useT()` to translate the titles. I'm open to other options, as long as the text that gets displayed to the user gets passed through transifex somehow.

Reply:
done